docs/AndroidBuildingFromSource.md

```diff
@@ -20,13 +20,15 @@ Make sure you have the following installed:
 3. Local Maven repository for Support Libraries (formerly `Android Support Repository`) >= 17 (for Android Support Library)
 4. Android NDK (download links and installation instructions below)
 
-Point Gradle to your Android SDK: either have `$ANDROID_SDK` and `$ANDROID_NDK ` defined, or create a local.properties file in the root of your react-native checkout with the following contents:
+Point Gradle to your Android SDK: either have `$ANDROID_SDK` and `$ANDROID_NDK ` defined in your bash profile, or create a local.properties file in the `android` directory of your react-native app with the following contents:
```

Comment:
@satya164 I dont't think both are required but I think its a good idea to have both setups in the docs for good measure. I had to spell out "Step 1" and "Step 2" because something was causing standard numerals not to increment properly.

Anywho, I think this new update makes things clearer. Thoughts?